### PR TITLE
Load button and refine design

### PR DESCRIPTION
**Pull request by hekmatnajafi**
added load buttons in main page and talent page
replace all module css with styled components in top seller page and talent page (index file)

### CODE REVIEW

File: pages/profile/[id]/index.js [hunk 1]
```diff
@@ -120,20 +112,22 @@ function Profile() {
...
+        </ProfileContainer>
         <Tabs defaultActiveKey="1" onChange={(e) => loadTabData(e)}>
           <TabPane tab="Created" key="1">
             {isLoading ?
                       <LoadingContainer>
                         <Spin />
                       </LoadingContainer>
-                      :
+                      : 
+                      <>
```
Comment (prince1456) [inline pages/profile/[id]/index.js:127]: please fix all indentation

File: pages/profile/[id]/index.js [hunk 2]
```diff
@@ -61,35 +59,29 @@ function Profile() {
...
-                <Col lg={8} md={6} sm={2} xs={0}></Col>
-                <Col lg={8} md={12} sm={20} xs={24}></Col>
-                <Col lg={8} md={6} sm={2} xs={0}></Col>
-              </Row>
-
               <div className="mt-4">
-                <button type="button" className={profileStyles.otherLinks}>
+                <ProfileButton type="button">
                   <svg
                     viewBox="0 0 15 16"
```
Comment (prince1456) [inline pages/profile/[id]/index.js:86]: Donot use SVG directly in the codes

File: pages/profile/[id]/index.js [hunk 3]
```diff
@@ -61,35 +59,29 @@ function Profile() {
     <>
       <Header />
       <div style={{ maxWidth: 1400, margin: "auto" }}>
```
Comment (prince1456) [inline pages/profile/[id]/index.js:61]: usually contianer should have maxSize: 1400 and padding left and right 20 pixel

Comment (prince1456) [reply pages/profile/[id]/index.js:61]: and if you make a common component for container and reuse it for other pages too it should be perfect

File: Components/topSellers.js [hunk 4]
```diff
@@ -40,42 +42,41 @@ function TopSellers({ data }) {
...
+        <>
           <div className="p-3">
             <h3>Top Sellers in 1 Day</h3>
           </div>
-          <div className={styles.topSellerContainer}>
+          <TopSellerContainer>
             {data &&
               data.map((seller, index) => (
                 //  <Link href='/profile/[id]' as={`/profile/${seller.address}`} >
                 <a href={`/profile/index?address=${seller.address}&talent=${seller.talent}&avatar=${seller.profile_img_url}`}>
```
Comment (prince1456) [inline Components/topSellers.js:53]: dont forget add key and fix indentation

**Review by prince1456 — CHANGES_REQUESTED** [review]

File: pages/profile/[id]/index.js (hunk 3 above)
Comment (hekmatnajafi) [reply pages/profile/[id]/index.js:61]: already did that and used, but forgot to use in talent page.

File: pages/profile/[id]/index.js [hunk 5]
```diff
@@ -60,94 +61,47 @@ function Profile() {
...
-            <div className={profileStyles.avatar}>
-              <img
-                alt="Identicon"
-                src={avatar}
-                loading="lazy"
-                className="sc-eirseW evgNzS"
-              />
+      <MainWrapper>
+        <ProfileContainer>
+          <div><img src="/images/talentCover.png" /></div>
```
Comment (prince1456) [inline pages/profile/[id]/index.js:66]: what is the reason for this div?
we should avoid to insert unnecceserly HTML element into DOM. for now its ok but keep in mind

File: pages/profile/[id]/index.js [hunk 6]
```diff
@@ -60,94 +61,47 @@ function Profile() {
...
-                        <Spin />
-                      </LoadingContainer>
-                      :
-                      <Products data={collections} /> 
-
-                      }
+            <LoadingContainer><Spin /></LoadingContainer>
+            :
+            <>
+            <Products data={collections} />
```
Comment (prince1456) [inline pages/profile/[id]/index.js:96]: always code should have correct indetation

**Review by prince1456 — APPROVED** [review]

File: pages/profile/[id]/index.js (hunk 5 above)
Comment (hekmatnajafi) [reply pages/profile/[id]/index.js:66]: mainwrapper? 
the div that has max width of 1400px with 20px paddings, i placed that in global styled component which can be used anywhere needed.

Comment (hekmatnajafi) [reply pages/profile/[id]/index.js]: and if you mean the div which has img inside it, i removed that

**Pull request merged**